jolt-core/src/jolt/vm/bytecode.rs

```diff
@@ -238,8 +233,21 @@ impl<F: PrimeField, G: CurveGroup<ScalarField = F>> BytecodePolynomials<F, G> {
     }
 
     fn preprocess(bytecode: &mut Vec<ELFRow>, trace: &mut Vec<ELFRow>) {
+        for instruction in bytecode.iter_mut() {
+            assert!(instruction.address >= RAM_START_ADDRESS as usize);
+            assert!(instruction.address % BYTES_PER_INSTRUCTION == 0);
```

Comment:
Do this need to be spaced by`BYTES_PER_INSTRUCTION`? Could we make them sequential to get some cost savings?

(Don't need to handle here, but good to discuss / track)

Reply:
it's handled two lines below:
```
instruction.address /= BYTES_PER_INSTRUCTION;
```